plugins/frontend/backstage-plugin-wiz/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@roadiehq/backstage-plugin-wiz",
-  "version": "1.0.4",
+  "version": "1.0.5",
```

Comment:
It is already published so no need for changeset

Reply:
What do you mean by this? Published from the CLI you mean?